docs/docsite/rst/become.rst

```diff
@@ -1,29 +1,32 @@
+*****************************
 Become (Privilege Escalation)
-+++++++++++++++++++++++++++++
+*****************************
 
 Ansible can use existing privilege escalation systems to allow a user to execute tasks as another.
 
 .. contents:: Topics
 
 Become
-``````
+======
+
 Ansible allows you to 'become' another user, different from the user that logged into the machine (remote user). This is done using existing privilege escalation tools such as `sudo`, `su`, `pfexec`, `doas`, `pbrun`, `dzdo`, `ksu`, `runas` and others.
 
 
 .. note:: Prior to version 1.9, Ansible mostly allowed the use of `sudo` and a limited use of `su` to allow a login/remote user to become a different user and execute tasks and create resources with the second user's permissions. As of Ansible version 1.9,  `become` supersedes the old sudo/su, while still being backwards compatible. This new implementation also makes it easier to add other privilege escalation tools, including `pbrun` (Powerbroker), `pfexec`, `dzdo` (Centrify), and others.
 
-.. note:: Become vars and directives are independent. For example, setting `become_user` does not set `become`.
+.. note:: Become vars and directives are independent. For example, setting ``become_user`` does not set ``become``.
 
 
 Directives
------------
+==========
+
 These can be set from play to task level, but are overridden by connection variables as they can be host specific.
 
 become
-    set to 'true'/'yes' to activate privilege escalation.
+    set to ``True`` (or ``Yes``) to activate privilege escalation.
```

Comment:
Why make this title-case ? I thought we standardized on 'yes' and 'no' for boolean values in examples and documentation ?

Reply:
Yup, we did. Thanks for reminding me